### PR TITLE
Closes #93 | Add dataloader for Khmer ALT POS

### LINKED ISSUE #93: Create dataset loader for Khmer ALT POS
Dataloader name: `khmer_alt_pos/khmer_alt_pos.py`
DataCatalogue: http://seacrowd.github.io/seacrowd-catalogue/card.html?khmer_alt_pos

|     Dataset| khmer_alt_pos |
|-------------|---|
| Description | The data contains 20,000-sentence Khmer corpus with manual tokenization and POS-tagging annotation. |
| Subsets | - |
| Languages | khm |
| Tasks | POS Tagging |
| License | Creative Commons Attribution Share Alike 4.0 (cc-by-sa-4.0) |
| Homepage | https://www2.nict.go.jp/astrec-att/member/mutiyama/ALT/km-nova-181101.zip |
| HF URL | - |
| Paper URL | https://dl.acm.org/doi/10.1145/3464378 |

### PR DESCRIPTION
**Pull request by muhsatrio**
Closes #93

### Checkbox
- [x] Confirm that this PR is linked to the dataset issue.
- [x] Create the dataloader script `seacrowd/sea_datasets/khmer_alt_pos/khmer_alt_pos.py` (please use only lowercase and underscore for dataset naming).
- [x] Provide values for the `_CITATION`, `_DATASETNAME`, `_DESCRIPTION`, `_HOMEPAGE`, `_LICENSE`, `_URLs`, `_SUPPORTED_TASKS`, `_SOURCE_VERSION`, and `_SEACROWD_VERSION` variables.
- [x] Implement `_info()`, `_split_generators()` and `_generate_examples()` in dataloader script.
- [x] Make sure that the `BUILDER_CONFIGS` class attribute is a list with at least one `SEACrowdConfig` for the source schema and one for a seacrowd schema.
- [x] Confirm dataloader script works with `datasets.load_dataset` function.
- [x] Confirm that your dataloader script passes the test suite run with `python -m tests.test_seacrowd seacrowd/sea_datasets/khmer_alt_pos/khmer_alt_pos.py`.

### CODE REVIEW

File: seacrowd/sea_datasets/khmer_alt_pos/khmer_alt_pos.py [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+
+_DATASETNAME = "khmer_alt_pos"
+
+_DESCRIPTION = """\
+The data contains 20,000-sentence Khmer corpus with manual tokenization and POS-tagging annotation.
+"""
+
+_HOMEPAGE = "https://www2.nict.go.jp/astrec-att/member/mutiyama/ALT/km-nova-181101/README.txt"
+
+_LICENSE = Licenses.CC_BY_4_0.value
```
Comment (ryanignatius) [inline seacrowd/sea_datasets/khmer_alt_pos/khmer_alt_pos.py:58]: from readme file in issue homepage,
I find the license is `Creative Commons Attribution-NonCommercial-ShareAlike 4.0 International (CC BY-NC-SA 4.0) License`
```suggestion
_LICENSE = Licenses.CC_BY_NC_SA_4_0.value
```

File: seacrowd/sea_datasets/khmer_alt_pos/khmer_alt_pos.py [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+            name=f"{_DATASETNAME}_source",
+            version=SOURCE_VERSION,
+            description=f"{_DATASETNAME} source schema",
+            schema="source",
+            subset_id=f"{_DATASETNAME}",
+        ),
+        SEACrowdConfig(
+            name=f"{_DATASETNAME}_seacrowd_seq_label",
+            version=SEACROWD_VERSION,
+            description=f"{_DATASETNAME} Nusantara Seq Label schema",
```
Comment (ryanignatius) [inline seacrowd/sea_datasets/khmer_alt_pos/khmer_alt_pos.py:88]: please change Nusantara with SEACrowd

**Comment by ryanignatius** [comment]
please add `__init__.py` file too

**Comment by muhsatrio** [comment]
Hi @ryanignatius, had adjusted as reviewed by you. Thank you!

**Review by ryanignatius — APPROVED** [review]
Reviewed and tested, LGTM

**Review by holylovenia — APPROVED** [review]
LGTM too!! Thanks for your help, @muhsatrio 😄

**Pull request merged**